infrastructure/scripts/provision/airy-core.sh

```diff
@@ -2,27 +2,27 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+app_image_tag="${APP_IMAGE_TAG:-latest}"
```

Comment:
the default should come from the outside I think (as in the bootstrap...?) otherwise it may be error prone